novatel_gps_msgs not built before novatel_gps_driver

When trying to build this package as part of a larger workspace using catkin_make, I got the following error repeatedly. 
```
In file included from /opt/carma/src/CARMANovatelGpsDriver/novatel_gps_driver/src/novatel_gps.cpp:35:0:
/opt/carma/src/CARMANovatelGpsDriver/novatel_gps_driver/include/novatel_gps_driver/novatel_gps.h:47:36: fatal error: novatel_gps_msgs/Gpgga.h: No such file or directory
compilation terminated.
CARMANovatelGpsDriver/novatel_gps_driver/CMakeFiles/novatel_gps_driver.dir/build.make:62: recipe for target 'CARMANovatelGpsDriver/novatel_gps_driver/CMakeFiles/novatel_gps_driver.dir/src/novatel_gps.cpp.o' failed
make[2]: *** [CARMANovatelGpsDriver/novatel_gps_driver/CMakeFiles/novatel_gps_driver.dir/src/novatel_gps.cpp.o] Error 1
CMakeFiles/Makefile2:22174: recipe for target 'CARMANovatelGpsDriver/novatel_gps_driver/CMakeFiles/novatel_gps_driver.dir/all' failed
make[1]: *** [CARMANovatelGpsDriver/novatel_gps_driver/CMakeFiles/novatel_gps_driver.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
```

This suggests to me that the novatel_gps_msgs package was not built before the novatel_gps_driver. I found that if I built the novatel_gps_msgs package on its own first, then the novatel_gps_driver would build without issue. 

My development environment is Ubuntu 16.04, ROS Kinetic, cmake 3.5.1, gcc 5.4.0, Python 2.7.12 and Python 3.5.2

--- a/novatel_gps_driver/CMakeLists.txt
+++ b/novatel_gps_driver/CMakeLists.txt
@@ -67,6 +67,9 @@ add_library(${PROJECT_NAME}
   src/parsers/time.cpp
   src/parsers/trackstat.cpp
 )
+
+add_dependencies(${PROJECT_NAME} ${catkin_EXPORTED_TARGETS})
+
 target_link_libraries(${PROJECT_NAME}
   ${catkin_LIBRARIES}
   ${Boost_LIBRARIES}

